core/src/main/resources/hudson/model/ParametersAction/index.jelly

```diff
@@ -35,12 +35,15 @@ THE SOFTWARE.
 		<st:include page="sidepanel.jelly" it="${build}" />
 		<l:main-panel>
 			<h1>${%Build} ${build.displayName}</h1>
-			<l:pane title="${%Parameters}" width="3">
-			<j:forEach var="parameterValue" items="${it.parameters}">
-				<st:include it="${parameterValue}"
-					page="value.jelly" />
-			</j:forEach>
-			</l:pane>
+			<p>${%Parameters} :</p>
```

Comment:
should this not be a heading or something - it seems to get lost now - whereas before it was styled differently